onnx/numpy_helper.py

```diff
@@ -281,6 +283,11 @@ def from_array(arr: np.ndarray, name: Optional[str] = None) -> TensorProto:
     Returns:
         TensorProto: the converted tensor def.
     """
+    if not isinstance(arr, np.ndarray) and not isinstance(arr, np.generic):
```

Comment:
nit: can be combined

```suggestion
    if not isinstance(arr, (np.ndarray, np.generic)):
```